utils/build/docker/java/spring-boot/src/main/java/com/datadoghq/system_tests/springboot/App.java

```diff
@@ -119,6 +119,11 @@ String headers(HttpServletResponse response) {
         return "012345678901234567890123456789012345678901";
     }
 
+    @GetMapping("/some-endpoint")
```

Comment:
Do we need to create a new endpoint? Can we just use the root path: `/` (on line 108). It returns the same value too

Reply:
Adding a new endpoint to all weblog apps is a pain and we should avoid it if we can

Reply:
+1